### PR TITLE
[RORDEV-1167] ROR caching details

### PR DESCRIPTION
**Pull request by coutoPL**
ref: https://github.com/beshu-tech/readonlyrest-docs/pull/206#discussion_r1580641548

### CODE REVIEW

**Comment by coutoPL** [comment]
@sscarduzio kindly reminder ;)

File: details/caching.md [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+
+* the block `Block 1` - "find all groups for the user that start with `group1_`"
+* the block `Block 2` - "find all groups for the user that start with `group2_`"
+* the block `Block 3` - "find all groups for the user that start with `group3_`"
+
+Oh yes. It should be obvious now. We don't get all user groups, just a subset of all existing groups. We do cache the information, but each block can't find the information it's looking for in the cache. So we call LDAP in each block. 
+
+Is it bad or not? It depends. On the business case, of course. In some cases it's better to enable server-side rendering (e.g. when the LDAP groups are a really long list), but in other cases it's better to reduce LDAP calls (so more data can be sent over the network). In ROR, server-side group filtering is disabled by default. You should consider whether it should be enabled in your case. But be aware of the caching implications described above.
```
Comment (sscarduzio) [inline details/caching.md:184]: server side filtering you mean? Got some React vibes here :)

Comment (sscarduzio) [inline details/caching.md:184]: Maybe let's hilight this consideration about when to use server side filtering in its own paragraph? Like "When to enable server-side groups filtering?"

**Comment by coutoPL** [comment]
@sscarduzio do you want me to add/change anything else?

**Review by sscarduzio — APPROVED** [review]

**Pull request merged**